lib/sensors/MoistureSensor.h

```diff
@@ -0,0 +1,20 @@
+#include "mbed.h"
+#include <I2C.h>
+
+#define I2C_Addr 0x36
+
+class MoistureSensor {
+    public:
+
+        MoistureSensor(PinName sda, PinName scl, char* data, int address = I2C_Addr);   //choose the right fucking pins
```

Comment:
no need to choose pins. the pinconfig is only relevant in the app. for the library, what u have rn is good!